medical-portal/src/UI/src/app/app.model.ts

```diff
@@ -12,3 +12,7 @@ export enum CaseStageEnum {
   'IntakeValidation' = 'Intake Validation',
   'Closed' = 'Closed',
 }
+
+export enum DocumentTypeEnum {
```

Comment:
small details but we don't use Hungarian naming convention. This can be called something like DocumentType